packages/manager/modules/support/src/tickets/new-ticket/new-ticket.html

```diff
@@ -93,10 +93,13 @@
     <p>
         <strong data-translate="ovhManagerSupport_new_creation_error"></strong>
     </p>
-    <p>
+    <p data-ng-if="$ctrl.isDedicatedServer()">
```

Comment:
Can we have something more generic? Tomorrow if we have to display a specific message for all products, do we plan to add multiple `p` with `ng-if` ?

Reply:
No there is a p for all other cases which display a global message

Reply:
Like it was initialy

Reply:
@mohammed-zahaf there is for other cases for now